pages/index.js

```diff
@@ -165,6 +138,7 @@ const STYLES_LINK = css`
 
 const STYLES_H1 = css`
   font-size: ${Constants.typescale.lvl5};
+  font-weight: 700;
```

Comment:
Don't use font-weight, use font-family and add the fonts you need.

Reply:
wait I thought you wanted us to add `font-weight: 400` yesterday? Is there a difference?